docs/Metrics-System.md

```diff
@@ -51,20 +51,36 @@ The following shows the details of the available metrics.
 * CapacityTotal: Total capacity of the file system in bytes.
 * CapacityUsed: Used capacity of the file system in bytes.
 * CapacityFree: Free capacity of the file system in bytes.
+* FilesPinned: Total number of the files pinned.
```

Comment:
what are the logical blocks in this documentation? (e.g. why is FilesPinned here as opposed to with other Files* methods)?